EIPS/eip-7727.md

```diff
@@ -0,0 +1,228 @@
+---
+eip: 7727
+title: EVM Transaction Bundles
+description: Enable EVM support for transaction bundles without revert protections.
+author: Lily Johnson (@lilyjjo)
+discussions-to: https://ethereum-magicians.org/t/eip-7727-evm-transaction-bundles/20322
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-06-24
+requires: 2718
+---
+
+## Abstract
+
+This EIP introduces two new [EIP-2718](./eip-2718.md) transaction types and one new opcode, enabling smart contracts and transactions to delegate their local sequencing rights to an off-chain entity. These new transaction types work together to create EVM-native 'bundles', which are similar but weaker than the PBS bundles offered by builders to searchers.
```

Comment:
```suggestion
This EIP introduces two new [EIP-2718](./eip-2718.md) transaction types and one new opcode, enabling smart contracts and transactions to delegate their local sequencing rights to an off-chain entity. These new transaction types work together to create EVM-native 'bundles', which are similar to but weaker than the Proposer-Builder Separation (PBS) bundles offered by builders to searchers.
```

I assume PBS expands to the above. I prefer to expand abbreviations the first time they are used.